ground/src/main/java/com/google/android/ground/ui/tos/TermsOfServiceViewModel.kt

```diff
@@ -56,13 +56,16 @@ constructor(
         }
       emit(Html.fromHtml(html, 0))
     } catch (e: Throwable) {
-      if (e !is TimeoutCancellationException && !e.isPermissionDeniedException()) {
+      if (e.isExpectedFailure()) {
```

Comment:
Don't we have to negate the condition?

Reply:
Ugh, you're right. Thanks for catching this!